### DISCUSSION TITLE
Add Glyph of Typhoon to glyph vendors

### DISCUSSION THREAD

**Discussion by trclocke** [chromiecraft/chromiecraft#5976] [Suggestions]
[Glyph of Typhoon](https://www.wowhead.com/wotlk/item=44922/glyph-of-typhoon) is a minor glyph that requires Northrend mats to craft (Ink of the Sea). The spell is available through talents from level 50, the glyph is usable at level 70. Seems reasonable that it be purchasable.

**Comment by QuantaCosta** [comment]
going with OP, might be a minor thing, but a nuisance if you wanna use it. No reason to not be obtainable.
